packages/js/core-client/src/PolywrapCoreClient.ts

```diff
@@ -18,72 +16,35 @@ import {
   UriResolutionContext,
   getEnvFromUriHistory,
   InvokeResult,
-  ValidateOptions,
   buildCleanUriHistory,
+  CoreClientConfig,
   WrapError,
   WrapErrorCode,
 } from "@polywrap/core-js";
 import { msgpackEncode, msgpackDecode } from "@polywrap/msgpack-js";
 import {
-  compareSignature,
   DeserializeManifestOptions,
   WrapManifest,
-  ImportedModuleDefinition,
 } from "@polywrap/wrap-manifest-types-js";
-import { Tracer, TracerConfig, TracingLevel } from "@polywrap/tracing-js";
```

Comment:
While I think it might be correct to removing tracing from the core client and core, I think it should be a separate PR so it can be evaluated in isolation